eq-author/src/App/questionPage/Design/QuestionProperties/index.test.js

```diff
@@ -39,4 +39,14 @@ describe("QuestionProperties", () => {
     wrapper.find(byTestAttr(id)).simulate("change", value);
     expect(onUpdatePage).toHaveBeenCalledWith({ foo: "bar", id: "1" });
   });
+
+  it("should correctly handle HelpButton onClick", () => {
+    wrapper.find(HelpButton).simulate("click");
+    expect(wrapper.find(HelpModal).prop("isOpen")).toBeTruthy();
+  });
+
+  it("should correctly handle HelpModal onClose", () => {
+    wrapper.find(HelpModal).simulate("close");
+    expect(wrapper.find(HelpModal).prop("isOpen")).toBeFalsy();
```

Comment:
Would it have been falsy before the close action as well?

Reply:
I've merged this into the previous test which now tests the opening / closing of the modal together 👍